carma-messenger-core/cpp_message/src/cpp_message.cpp

```diff
@@ -1558,7 +1657,9 @@ namespace cpp_message
             Id64b_t* id64;
             id64 = (Id64b_t*)calloc(1, sizeof(Id64b_t));
 
-            uint8_t id_content[8];
+            uint8_t* id_content;
```

Comment:
How is this being freed?

Reply:
I didn't fix those, they're technically part of the TCR rather than the TCM. I could also fix it in this pr, it's like 1 hr to fix all